src/test/scala/org/camunda/feel/impl/builtin/BuiltinConversionFunctionsTest.scala

```diff
@@ -63,6 +63,18 @@ class BuiltinConversionFunctionsTest
     )
   }
 
+  it should "return null if the date is not valid (leap year)" in {
+
+    evaluateExpression(""" date(x) """, Map("x" -> "2023-02-29")) should returnNull()
```

Comment:
🔧 2023-02-29 is invalid because it is **not a leap year**, but the test description imply otherwise.

```suggestion
  it should "return null if the date is not valid (not a leap year)" in {

    evaluateExpression(""" date(x) """, Map("x" -> "2023-02-29")) should returnNull()
```